python_modules/dagster/dagster/_daemon/monitoring/monitoring_daemon.py

```diff
@@ -55,6 +55,47 @@ def monitor_starting_run(
         instance.report_run_failed(run, msg)
 
 
+def monitor_canceling_run(
+    instance: DagsterInstance, run_record: RunRecord, logger: logging.Logger
+) -> None:
+    run = run_record.dagster_run
```

Comment:
i'm curious in what cases the `update_timestamp` on the run record doesnt match what we calculate via the event lookup